Makefile

```
@@ -131,11 +140,36 @@ lint-static-check:
 lint: lint-static-check
 	$(LINT) run --timeout 5m
 
+.PHONY: multimod-verify
+multimod-verify: install-tools
+	@echo "Validating versions.yaml"
+	multimod verify
+
+COREPATH ?= "https://github.com/aws-observability/aws-otel-collector.git"
+.PHONY: multimod-sync-upstream
+multimod-sync-upstream: | $(MULTIMOD)
+	@[ ! -d $COREPATH ] || ( echo ">> Path to core repository must be set in COREPATH and must exist"; exit 1 )
+	$(MULTIMOD) verify && $(TOOLS_MOD_DIR) sync -a -o ${COREPATH}
+
+MODSET = adot-base adot-tools
+
+.PHONY: multimod-prerelease
+multimod-prerelease: | $(MULTIMOD)
+    @[ "${MODSET}" ] || ( echo ">> env var MODSET is not set"; exit 1 )
+    $(MULTIMOD) verify && $(MULTIMOD) prerelease -m ${MODSET}
+
+.PHONY: multimod-tag
+multimod-tag: | $(MULTIMOD)
+    @[ "${MODSET}" ] || ( echo ">> env var MODSET is not set"; exit 1 )
+    $(MULTIMOD) verify && $(MULTIMOD) tag -m ${MODSET} -c ${COMMIT}
+
 .PHONY: install-tools
 install-tools:
-	cd tools/linters && GOBIN=$(PWD)/bin go install golang.org/x/tools/cmd/goimports
-	cd tools/linters && GOBIN=$(PWD)/bin go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.0
-	cd tools/linters && GOBIN=$(PWD)/bin go install honnef.co/go/tools/cmd/staticcheck@v0.2.0
+	cd $(TOOLS_MOD_DIR) && go install go.opentelemetry.io/build-tools/multimod
```

```suggestion
	cd $(TOOLS_MOD_DIR) && GOBIN=$(PWD)/bin go install go.opentelemetry.io/build-tools/multimod
```

This should be installed in the same location as the other tools.  That said, I think if the other changes here are applied then this will be unused.

---

Yesterday, for some reasons, I ran this code locally but it turns out normal; however, when running in the github-env I was not able to run it because of not recognizing the multi-mod command. Therefore, I change this as a base to run in the github-env first.

---

It works now. As you suggested, should build the binaries first instead of install them asyn